lua/carbon-now.lua

```diff
@@ -91,37 +80,46 @@ local function get_open_command()
   if vim.fn.has("win32") then
     return "start"
   end
+
+  error("Couldn't find a launch command")
```

Comment:
```suggestion
  vim.api.nvim_err_writeln("[carbon-now.nvim] Couldn't find a launch command")
```

Reply:
Just googled it, you're right, good catch. I didn't know the difference between them. I really want to thank you for your time and feedbacks, for real yo' ! 🏆 💪🏼  

and yes i didn't forgot it, i will fix it on the next commits 🙌🏼

Reply:
Fixed on [8aa5de1](https://github.com/ellisonleao/carbon-now.nvim/pull/4/commits/8aa5de1d050c41f69bdedd20aebaff80a4f43313)